api/alerts.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 type Alert struct {
-	ID                 string   `graphql:"id"                 yaml:"-"                     json:"id"`
+	ID                 string   `graphql:"id"                 yaml:"-"                     json:"-"`
```

Comment:
I think that we should just leave the json as it is, even though it is inconsistent with the yaml.

Reply:
Shouldn't the run-as-user also be output, or are you saving that for when adding the query ownership?

Reply:
I'll leave the json as is 👍 

Yes, the userId will be emitted with queryOwnership later.